test/ArrayList.test.js

```diff
@@ -0,0 +1,282 @@
+const { assert } = require("chai");
+let ArrayList = require("../src/ArrayList");
+
+describe('ArrayList', () => {
+    describe('ArrayList', () => {
+        it('should return an object', () => {
+            const list = new ArrayList();
+    
+            assert.isObject(list);
+        });
+    });
+    
+    describe('ArrayList.clear()', () => {
+        it('should return size = 0 (undefined)', () => {
+            const list = new ArrayList();
+            const expectedSize = 0;
+            const expectedString = '[]';
+    
+            list.clear();
+    
+            assert.deepEqual(list.getSize(), expectedSize);
+            assert.deepEqual(list.getString(), expectedString);
+        });
+    
+        it('should return size = []', () => {
+            const list = new ArrayList();
+            const expectedSize = 0;
+            const expectedString = '[]';
+    
+            list.clear();
+    
+            assert.deepEqual(list.getSize(), expectedSize);
+            assert.deepEqual(list.getString(), expectedString);
+        });
+    
+        it('should expected [1]', () => {
+            const list = new ArrayList();
+            const value = [1];
+            const expectedString = '[]';
+            const expectedSize = 1;
+            const actual = list.getString();
```

Comment:
I have already written to your teammate: how can you get actual before a test action?